app/styles/styles.css

```diff
@@ -135,3 +140,30 @@ body {
   background-color: lightblue; 
   max-width: 100%;
 }
+
+.character-block {
+  display: inline-block;
+  width: 3em;
+  height: 3em;
+  margin: .2em;
+  vertical-align: top;
+  text-align: center;
+}
+.character-block span {
+  font-size: 2em;
+  padding: 0.3em 0.4em;
+}
+
+.is-letter {
+  box-shadow: 0 .1em 0 0 black;
+}
+
+.grave-letter {
+  display: inline-block;
+  margin: .2em;
+  vertical-align: top;
+  font-size: 2em;
+  padding: 0.3em 0.4em;
+  border: 1px solid rgb(0,0,0);
+  border-radius: 100% 100% 0 0 ;
+}
```

Comment:
They should be in order of appearance. I'm not in charge of overall styles, however, so this is subject to change.